validator_go117_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build go1.17
-// +build go1.17
+//go:build go1.17 && !go1.20
+// +build go1.17,!go1.20
```

Comment:
Do we not need a similar thing in the `validator_go116_test.go` file?

Reply:
It has `!go1.17` which means only versions prior to 1.17.